source/include/shadow.h

```diff
@@ -1,4 +1,5 @@
 /*
+ * Device Shadow Library V1.0.0
```

Comment:
Is this the right name? Should it be `AWS IoT Device Shadow Library V1.0.0`?

Reply:
This is a PM question

Reply:
according to Tanmoy:  AWS IoT Device Shadow V1.0.0